 Fix NPE crash while removeBatch

When unpairing two devices, the BluetoothOppShareInfo
in BluetoothOppBatch may be null during transfer.

diff --git a/src/com/android/bluetooth/opp/BluetoothOppService.java b/src/com/android/bluetooth/opp/BluetoothOppService.java
index 040d2b6..00fc5fa 100644
--- a/src/com/android/bluetooth/opp/BluetoothOppService.java
+++ b/src/com/android/bluetooth/opp/BluetoothOppService.java
@@ -944,7 +944,8 @@ private void removeBatch(BluetoothOppBatch batch) {
                         mServerTransfer = new BluetoothOppTransfer(this, mPowerManager, nextBatch,
                                                                    mServerSession);
                         mServerTransfer.start();
-                        if (nextBatch.getPendingShare().mConfirm ==
+                        if (nextBatch.getPendingShare() != null
+                            && nextBatch.getPendingShare().mConfirm ==
                                 BluetoothShare.USER_CONFIRMATION_CONFIRMED) {
                             mServerTransfer.confirmStatusChanged();
                         }
